frontend/src/scenes/funnels/funnelLogic.js

```diff
@@ -67,20 +68,36 @@ export const funnelLogic = kea({
                 if (props.cachedResults && !refresh && values.filters === props.filters) {
                     return props.cachedResults
                 }
+
                 const { from_dashboard } = values.filters
                 const cleanedParams = cleanFunnelParams(values.filters)
                 const params = {
                     ...(refresh ? { refresh: true } : {}),
                     ...(from_dashboard ? { from_dashboard } : {}),
                     ...cleanedParams,
                 }
+
                 let result
 
                 insightLogic.actions.startQuery()
+
+                const eventName = 'calculate funnel result'
+                const eventProps = {
+                    'event count': params.events.length,
+                    'action count': params.actions.length,
+                    'total count actions and events': params.actions.length + params.events.length,
+                    interval: params.interval,
+                }
+
                 try {
                     result = await pollFunnel(params)
+                    eventProps.success = true
+                    posthog.capture(eventName, eventProps)
```

Comment:
I think it would be better to have this logic in `eventUsageLogic.ts`, we've been centralizing the logic of custom event reporting in that module, makes it easier to maintain and also separates the core logic from event reporting logic.

Reply:
+1 I missed this but we should definitely keep analytics logic in `eventUsageLogic.ts`